client/src/features/user/userApi.ts

```diff
@@ -0,0 +1,60 @@
+import { Avatar, User, UserApi } from '../../types';
+import { gql } from '@apollo/client';
+import client from '../../api/graphql';
+import { getCookie } from '../../utils';
+
+export const userApi: UserApi = {
+  async getAvatar(): Promise<Avatar> {
+    try {
+      const jwt = getCookie('pizza-delivery-user-jwt');
+
+      const response = await client.query({
+        query: gql`
```

Comment:
Such things as going after `query: ...` - better to move to separate file - these things called `endpoints` - so then it will look like `query: AAA`